src/components/SearchBar/SearchBar.tsx

```diff
@@ -20,10 +20,7 @@ export interface SearchBarProps {
   sx?: object;
 }
 
-const SearchBar: React.ForwardRefRenderFunction<
-  HTMLDivElement,
-  SearchBarProps
-> = (props) => {
```

Comment:
I've remove some code here, because it was giving me error in stories, while I was trying to use mandatory props. Maybe somebody else has some objections, but decided to go to simpler way.